### ISSUE TITLE
More clarity on prerequisites (or package them in the appimage)

### ISSUE THREAD

**Issue by bernd-wechner** [lettier/gifcurry#76]
I just tried the appimage. And it says _"gtksink" not found. No video preview. Install all Gstreamer 1.0 plugins._

Alas the readme is not very help stating only: Make sure you have FFmpeg, GStreamer, ImageMagick, and GTK+ installed on your machine.

Sure. Will do. I'm not even completely clueless in this space. Butt still, I'm on an Ubuntu box and the plugins are packaged in four groups, base, good, bad, ugly and I have them all installed. 

```
$ apt search gstreamer1.0-plugin | grep ^i
i A gstreamer1.0-plugins-bad                                                                    - GStreamer plugins from the "bad" set                                                                 
i   gstreamer1.0-plugins-base                                                                   - GStreamer plugins from the "base" set                                                                
i   gstreamer1.0-plugins-base-apps                                                              - GStreamer helper programs from the "base" set                                                        
i   gstreamer1.0-plugins-good                                                                   - GStreamer plugins from the "good" set                                                                
i A gstreamer1.0-plugins-ugly                                                                   - GStreamer plugins from the "ugly" set     
```

And no package seems to provide it:

```
$ apt-file search gtksink
gstreamer1.0-plugins-good-doc: /usr/share/gtk-doc/html/gst-plugins-good-plugins-1.0/gst-plugins-good-plugins-gtksink.html
```

So only a help file suggesting it's in the good plugins, which are installed. And yet indeed, `which`, and `locate` can't find it. Go figure.

The joy of appimage of course is that you can package all that, not? Just curious.

**Comment by bernd-wechner** [comment]
I actually found it thanks to: https://github.com/lettier/gifcurry/issues/23

But this issue stands. it would be a smoother experience if it was bundled in the appimage.

**Comment by henk717** [comment]
Had to do some digging as well since i was certain it just worked last time i used it but that was a while ago and i think i used a different linux ISO back then.
Turns out, for arch the package name is gst-plugin-gtk and its not included in the default plugin set which it probably should be.

**Comment by Uup115** [comment]
> I actually found it thanks to: #23
> 
> But this issue stands. it would be a smoother experience if it was bundled in the appimage.

Doesn't work for [me](https://github.com/lettier/gifcurry/issues/83#issue-1192319759)...

**Comment by scarlion1** [comment]
Yeah doesn't work for me either.  `gstreamer1.0-gtk3` 1.20.3 installed `/usr/lib/x86_64-linux-gnu/gstreamer-1.0/libgstgtk.so` and gifcurry isn't finding it I guess.

**Comment by 0lm** [comment]
Any walkaround by now? Im on Ubuntu 22.04.1 LTS on WSL2. Still cant find gtksink.
